src/ol/render/canvas/canvasreplay.js

```diff
@@ -267,10 +267,10 @@ ol.render.canvas.Replay.prototype.replay_ = function(
         if ((skipFeatures &&
             skippedFeaturesHash[goog.getUid(feature).toString()]) ||
             !feature.getGeometry()) {
-          i = /** @type {number} */ (instruction[2]);
+          i = /** @type {number} */ (instruction[2]) + 1;
```

Comment:
I think this change might break things with the Select interaction. But the one below is fine.